query_ctrl.js

```diff
@@ -27,7 +27,7 @@ function (angular, _, sdk) {
     KairosDBQueryCtrl.prototype = Object.create(_super.prototype);
     KairosDBQueryCtrl.prototype.constructor = KairosDBQueryCtrl;
 
-    KairosDBQueryCtrl.templateUrl = 'public/plugins/kairosdb/partials/query.editor.html';
+    KairosDBQueryCtrl.templateUrl = 'public/plugins/grafana-kairosdb-datasource/partials/query.editor.html';
```

Comment:
If that will work it might be a good idea to make that the preferred syntax and deprecate urls with the full plugin path embedded in them.